themes/vscode/vscode.theme.sh

```diff
@@ -5,7 +5,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-_omb_theme_PROMPT_COMMAND() {
+_omb_theme_vscode_initialize() {
```

Comment:
The function name `_omb_theme_PROMPT_COMMAND` is used for the function that is supposed to be called after every execution of a user command. This function seems to be only executed once in the initialization time and soon delete by `unset -f`, so the name does not match with the function purpose. Let me change the function name.